app/views/bills/index.html.erb

```diff
@@ -32,5 +32,21 @@
       </select>
     </div>
   </div>
+  <div class="row">
+    <div class="col-6">
+      <h4>Five recent bills:</h4>
+      <% @five_recent.each do |info| %>
+        <p id="pId">Bill ID: <a href="<%= info.congress_url %>: void(0)"><%= info.bill_id %></a></p>
+        <p>Bill Subject: <%= info.primary_subject %></p>
```

Comment:
Can we add the short_title as a link to the bill's congress_url, too? As a user, I think it would be helpful to see the title and get an idea of what the bill is about. Also, could we update "Bill ID" to "Bill #" or "Bill Number" to make an explicit connection to the bill number search above?